Makefile

```
@@ -202,6 +203,11 @@ check-envtest-existing:
 	USE_EXISTING_CLUSTER=true $(GO_TEST) -count=1 -cover -p=1 -tags=envtest ./...
 	${PGO_KUBE_CLIENT} delete -k ./config/dev
 
+# Expects operator to be running
+.PHONY: check-kuttl
+check-kuttl:
```

Perhaps

```suggestion
check-e2e:
```

Unless we're planning on adding other e2e testing libs?

---

I was leaving it open to add other e2e libs but I don't have any in mind at the moment. I was also following what we did for `envtest` with `make check-envtest{-existing}`

---

I like leaving it at check-kuttl this will allow us to add more e2e libs in the future and we could potentially add a check-e2e to then run all of them.

---

OK. Suggestion withdrawn.